lib/enum/basic_enum.rb

```diff
@@ -85,66 +98,86 @@ def element_frequency_count
   # http://ruby-doc.org/core-2.1.4/Enumerable.html#method-i-find-all
   # http://ruby-doc.org/core-2.1.4/Enumerable.html#method-i-select
   def elements_ending_in_er
+    collection.find_all { |element| element.include? "er" }
   end
 
 
   # Refer to find_index
   # http://ruby-doc.org/core-2.1.4/Enumerable.html#method-i-find-index
   def index_of_first_awesome_element
+    collection.find_index { |i| i[:awesome] == true }
```

Comment:
Since the collection consists of people, it's a good idea to name the local variable something that makes sense, like `|person|`